connectors/src/connectors/github/lib/github_api.ts

```diff
@@ -906,7 +902,7 @@ export async function processRepository({
             )}`,
             internalId: p.internalId,
             parentInternalId: dirParentInternalId,
-            parents: parents.slice(0, i).map((p) => p.internalId),
```

Comment:
just to be clear: this line is the real PR, the rest is a refactoring to avoid reverse (which is :+1: ) correct?

Reply:
yes, right on 👍